django/core/checks/security/base.py

```diff
@@ -107,6 +108,12 @@
     id='security.W021',
 )
 
+W022 = Warning(
+    "Your SECRET_KEY contains an invalid string or UTF-8 bytes. "
```

Comment:
This needs clarification about what an "invalid string" means. It could probably be a single sentence if you phrase it like "SECRET_KEY must be ..."